accepted/global-json-updates.md

```diff
@@ -0,0 +1,369 @@
+      * Otherwise, if there higher major versions, use the lowest higher major version in that set, its lowest minor, its lowest feature band, and the highest patch within that feature band. 
+      * Otherwise, fail.
+    * `latestPatch`
+      * Even if the requested major/minor/feature band/patch is found, do not use it unless it matches rules below.
+      * Considering only the equal and higher patches in the major/minor/feature band, select the highest available patch.
+      * If there are no equal or higher patches in the major/minor/feature band, fail.
+    * `latestFeature`
+      * Even if the requested major/minor/feature band is found, do not use it unless it matches rules below.
+      * Considering only the equal and higher feature bands in the major/minor, select the highest available feature band and its highest patch.
+      * If there are no equal or higher feature bands in the major/minor, fail.
+    * `latestMinor`
+      * Even if the requested major/minor is found, do not use it unless it matches rules below.
+      * Considering only the equal and higher minor versions in the major version, select the highest available minor, its highest feature band and its highest patch.
+      * If there are no equal or higher minor versions in the major, fail.
+    * `latestMajor` (latest)
+      * Even if the requested major is found, do not use it unless it matches rules below.
+      * Considering only the equal and higher major versions, select the highest available major, its highest minor, its highest feature band and its highest patch.
+      * If there are no equal or higher major versions, fail.
+    * `disable`
+      * Do not roll forward. Only bind to specified version. This policy means the latest patches, including security patches will not be used.
+ 
+## Default
+
+### No `global.json`
+
+If there is no `global.json` or a version is not specified, the default is`latestMajor`, again to match today's behavior.
+
+### `global.json` containing a version number
+
+**Breaking Change** This default represents a breaking change. We took this change because we believe the new behavior will cause no change or be an improvement.  
+
+Previously, if the specified patch version was present, it was used even if a higher patch number was installed. In discussing this with users, ignoring the updated patch was not the anticipated behavior. Also, the previous default did not roll forward across feature, minor or major bands. This could result in failure when a usable SDK existed on the machine.
+
+The new behavior is that the highest patch within the feature band will be used even if the requested version is available, unless the `rollForward` is set to `disable`.
+
+Also, the nearest higher SDK will be used if there is no version that matches the requested major, minor and feature. This behavior can be changed by specifying other options for the `rollForward` property. 
```

Comment:
Is it nearest higher SDK or is it basically RollForward=major by default? The difference I see is in the case where there are multiple patches.

Reply:
I think the default should map to some well defined roll forward setting - it should not be yet another way of doing things - that's super confusing. I don't know which is the best default... currently it is basically `patch`. Changing it to `major` feels like a rather big change. But I guess we make different guarantees about SDK backward compatibility across major versions than we do for the product...

Reply:
VS will be cleaning up after itself moving forward and will only have a current SDK with optional runtimes, templates, etc. for older version. As such, using recent SDK with projects targeting older runtimes is mainline scenario. We should not break it as a general rule. There will be exceptions and those can be handled with an explicit strengthening in global.json IMHO. The current strictness by default causes a mess of having to keep a bazillion sdks to open up different projects. And we're not going to be leaving SDKs behind anymore.

Reply:
> There will be exceptions and those can be handled with an explicit strengthening in global.json IMHO.

Or you can install older SDK and major will still prefer it. In the case where you hit a break. 

I do think that the sdk compatibility is different than runtime compatibility. We have always (meaning before .NET Core even) given customers latest tools (Visual Studio et al) to use on their existing projects. I think this should always work by default.